client/packages/system/src/Location/ListView/ListView.tsx

```diff
@@ -38,6 +39,7 @@ const LocationListComponent: FC = () => {
   const t = useTranslation();
   const columns = useColumns<LocationRowFragment>(
     [
+      GenericColumnKey.Selection, // adds padding to the left not sure why
```

Comment:
Just a bit unsure which padding in the left we're referring to. Would it be the green or the blue one?
![Screenshot 2024-12-16 at 8 41 29 AM](https://github.com/user-attachments/assets/bbe8799c-54ba-48b8-a2c7-3c3270ddaa14)

Reply:
@GeronimoJohn thanks for checking! It doesn't add padding to all columns. It's only visible in my second screenshot (Stocktakes) and my last screenshot (Inventory Locations).

Reply:
It's not padding... but when the screen gets wider, it needs somewhere to put the extra width.

If we have some columns with set widths, and some with none, then the extra width only goes on the columns with no set width, because they can grow as much as needed. (This is usually the case)

Checkbox does have width, so usually stays fixed at 60:

https://github.com/msupply-foundation/open-msupply/blob/5b2aecdc71c6c48902b3524b6e9b2f0cda04a75c/client/packages/common/src/ui/layout/tables/columns/CheckboxSelectionColumn.tsx#L34-L40

 However, if _all_ the columns have a defined width (a lot of them are defined in `ColumnDefinitionSetBuilder` if you're curious 😅) then it shares the extra space between all the columns. 


I guess we didn't notice this when select was on the end, because of the `align: ColumnAlign.Right`. Would be an improvement to make it `align: ColumnAlign.Left,` now!
![Screenshot 2024-12-16 at 4 34 00 PM](https://github.com/user-attachments/assets/27e3bc8f-f4a6-4e15-9585-2df7d7951b63)

Potentially, could also remove the `width` property from some columns of those two tables that are playing up, so only the columns we actually want to grow will grow 🤷‍♀️

Reply:
Right sorry misread that. Comment makes sense!

I think I found where the issue is happening. The component `getCheckboxSelectionColumn`. Sets the check box's alignment to be on the right. 

![Screenshot 2024-12-16 at 4 43 43 PM](https://github.com/user-attachments/assets/8d2b4eb8-3f86-403a-9f37-dfed920d5990)

Changing it to left seems to fix the issue. 

![Screenshot 2024-12-16 at 4 46 28 PM](https://github.com/user-attachments/assets/9095aa65-fd69-4eb2-8ac8-2e7d6d009d98)

The extra spacing are either being caused by Material UI or just how we've set up `DataTable`. Regardless of how the width is set, `ColumnAlign.Right` just uses that given space and yea puts it on right.

IMO we should probably move the alignment to left now. 😃

Reply:
Thank you both! Updated here 17be43b6324e0a82409a1a5e4244ac890a4e219e :)

Reply:
Hahaha didn't see Laché's comment